src/web3client/liquidator.js

```diff
@@ -11,6 +11,13 @@ class Liquidator {
         this.app.logger.info(`app.shutdown() - closing liquidation`);
         return;
       }
+
+      if(this.app.isRPCDrifting()) {
+        return {
+          error: "app.start() - RPC drifting",
```

Comment:
shouldn't this be "liquidator.start() ..."?

Reply:
done